helm_deploy/values-research.yaml

```diff
@@ -22,4 +22,4 @@ env:
   COMMUNITYAPI_BASEURL: "https://community-api-secure.test.delius.probation.hmpps.dsd.io"
   ASSESSRISKSANDNEEDS_BASEURL: "https://assess-risks-and-needs-dev.hmpps.service.justice.gov.uk"
   SENTRY_ENVIRONMENT: "research"
-  APPOINTMENTS_BOOKINGS_ENABLED: false
+  APPOINTMENTS_BOOKINGS_ENABLED: true
```

Comment:
_nit_ could also be removed

Reply:
Will raise a ticket to remove the feature and include this